chebyshev_distance.go

```diff
@@ -0,0 +1,34 @@
+//
+// Compiled with Go version go1.9 linux/amd64
+// Contributor : Shivendra Mishra 
+// Contact : shivendra_mishra@live.com
+//
+package stats
+
+import (
+	"math"
+)
+
+// Compute Chebyshev distance between two data points 
+func ComputeChebyshevDistance( data_point_x, data_point_y []float64 ) ( distance float64, 
```

Comment:
The spaces are not idiomatic or consistent with the rest of the codebase. `go fmt` will fix these :)